app/javascript/orangelight/vue_components/bookmark_button.vue

```diff
@@ -110,6 +110,7 @@ function toggle() {
   display: flex;
   align-items: center;
   border: 0.125rem solid var(--color-grayscale-light);
+  margin-bottom: 8px;
```

Comment:
We could also use this variable, which is currently defined as 8px

```suggestion
  margin-bottom: var(--space-x-small);
```